static/builder/dataStorage.js

```diff
@@ -478,6 +483,12 @@ class DataStorage {
         if (this.includeTrialRewards && totalNumber == 0 && item.access.includes("trial")) {
             totalNumber += 1;
         }
+        if (this.includeEasilyObtainableItems && totalNumber == 0 && item.access.some(type => DATA_STORAGE_EASILY_OBTAINABLE_ITEMS.includes(type))) {
+            totalNumber += 1;
```

Comment:
You may want to add the item maxNumber if defined, 4 else. I added 1 for trial items because they are all limited to 1, but in fact I should have added item.maxNumber, and verified that all trial items has a maxNumber of 1.

Reply:
Nice, was thinking whether I should always default to 10, but realized that some chocobo items cap at 1, etc. Didn't know about maxNumber, adding now!